spec/services/export/report_spec.rb

```diff
@@ -286,3 +288,102 @@ def link_for_row(row)
     row[@headers_for_report.length + 7]
   end
 end
+
+# This entire spec is for when there aren't forecasts but there are activities -- it's essentially
+# a copy of the setup for the above but we never add a value to the forecast.
+RSpec.describe Export::Report do
+  before(:all) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.start
+
+    financial_quarter = FinancialQuarter.for_date(Date.parse("31-Mar-2022"))
+    financial_year = FinancialYear.for_date(Date.parse("31-Mar-2022"))
+
+    @report = create(
+      :report,
+      financial_quarter: financial_quarter.to_i,
+      financial_year: financial_year.to_i
+    )
+
+    @project = create(
+      :project_activity_with_implementing_organisations,
+      implementing_organisations_count: 2
+    )
+
+    @implementing_organisation =
+      create(
+        :implementing_organisation,
+        name: "The name of the organisation that implements the activity",
+        iati_reference: "IMP-002",
+        organisation_type: "10"
+      )
+
+    @third_party_project =
+      create(
+        :third_party_project_activity,
+        parent: @project
+      ).tap do |project|
+        project.implementing_organisations << @implementing_organisation
+      end
+
+    @actual_spend =
+      create(
+        :actual,
+        parent_activity_id: @project.id,
+        report: @report,
+        financial_quarter: @report.financial_quarter,
+        financial_year: @report.financial_year
+      )
+
+    previous_report = create(
+      :report,
+      financial_quarter: financial_quarter.pred.to_i,
+      financial_year: financial_year.to_i
+    )
+
+    @forecast =
```

Comment:
This is just from a first pass over this PR and not an in-depth review: do we need to setup this `@forecast` at all?